manifest.yml.tpl

```diff
@@ -10,6 +10,7 @@ applications:
 
   routes:
     - route: notify-sms-provider-stub-{{CF_SPACE}}.cloudapps.digital
```

Comment:
Do we still need this one?

Reply:
I'll leave it in for now so the app works on both routes. We can then do a seamless switch before removing it.